cmd/helm/inspect.go

```diff
@@ -256,9 +255,18 @@ func (i *inspectCmd) run() error {
 
 func findReadme(files []*any.Any) (file *any.Any) {
 	for _, file := range files {
-		if slice.ContainsString(readmeFileNames, strings.ToLower(file.TypeUrl), nil) {
+		if contains(readmeFileNames, strings.ToLower(file.TypeUrl)) {
 			return file
 		}
 	}
 	return nil
 }
+
+func contains(slice []string, s string) bool {
```

Comment:
We should just mimic `slice.ContainsString` since we rely on the modifier function. https://github.com/kubernetes/kubernetes/blob/3f47b4febdc81edcfcc2f36b60fc7062a2076911/pkg/util/slice/slice.go#L58-L70

Reply:
@bacongobbler Sure! I see  that this has been removed in dev-v3. Shall I just mimic that instead?

Reply:
Nah, I'd keep it in for Helm 2.